src/common/constants.ts

```diff
@@ -37,6 +37,13 @@ const VALID_TIMES = [
 
 const UNDECLARED_MAJOR = 'Undeclared';
 
+// ENVIRONMENT TYPE
+
+// Uses SSL
+const IS_PRODUCTION = process.env.NODE_PROD_ENV === 'true';
+// Allows access to dev routes
+const IS_DEVELOPMENT = process.env.NODE_DEV_ENV === 'true';
```

Comment:
I don't like how it's possible to have both `DEV` and `PROD` settings set `true`. Any given instance that's running should be either or explicitly. 
I propose we change the variable to `NODE_ENV` and check for it to either be `"PROD"` or `"DEV"`.

Reply:
@alanna-zhou and I decided on this change because in the codebase dev routes are only allowed to be used if on development. Allowing `DEV` and `PROD` to both be true would allow the dev routes to be used on the server, which would be helpful to check for issues that arise on production. For example, right now we've been having an issue with the server where the server crashes whenever someone tries to get all the users and we've been trying to access authenticated routes but we cannot because either a) we need to ask someone on ios (say @lucyxubroad ) to give us a google ID to authenticate and get an access token or b) we need to create a dummy user to use that access token but cannot because we're on production

but i guess since this is the server for testflight technically it should be on development and realistically we shouldn't need to be looking for bugs on the production server since we'd assume they were taken care of on development...
@alanna-zhou thoughts?

Reply:
it's not all black and white -- we used to just have PROD true, but the fact of the matter is that there are 3 different development scenarios: local, staging/dev, prod; the way that appdev has been doing is extremely non-scalable and we need to have different environments to properly debug on production-like environments. so, what gonzalo has here is a good solution so far (we also need to just let him merge please, we're trying to figure out a bug the server and this is blocking)